README.md

```diff
@@ -44,256 +44,207 @@ using image editing and stable diffusion models.**
+NEXT_PUBLIC_WS_SERVER_URL=ws://localhost:3002
+NEXT_PUBLIC_GOOGLE_ANALYTICS_ID=G-XXXXXXXXXX
+FAST_REFRESH=false
 ```
+3. Update the values with your own.
+4. Save the file.
 
-You only need to change the information in yaml file in order to update the information in db.
-
-#### MagicPrompt model
+Once you have updated the `.env.local` file, the client application will be able to connect to the backend API, Firebase, and other services using the values that you provided.
 
-In order to use MagicPrompt **locally**, you need to download and upload the model to the S3 bucket:
+**Step 7. Run the Morpheus project**
 
+To run the Morpheus project, execute the following command:
 ```bash
-# register models specified in morpheus-server/scripts/models/magicprompt-models-info.yaml
-docker compose run --rm model-script upload local magicprompt
+docker compose up
 ```
 
-MagicPrompt model will also be downloaded to the directory `morpheus-server/tmp`. If you want to add a new model, you
-only
-need to add its information in the
-file [magicprompt-models-info.yaml](./morpheus-server/scripts/models/magicprompt-models-info.yaml)
-
-For more information about this script, you can read the [README](./morpheus-server/scripts/models/README.md).
+The Morpheus project will be running on your local machine at localhost:3000 (client), localhost:3001 (admin), and
+localhost:8001 (api). Morpheus use an model for default, you can change puede hacerlo desde el panel de administración
 
-## Development
+# Development
 
-### Running the backend tests
+## Running the backend tests
+* To run all the tests: `docker compose run --rm api pytest`.
+* To run a specific test: `docker compose run --rm api pytest tests/«test_module».py`.
+* To run a specific test function: `docker compose run --rm api pytest tests/«test_module».py::«test_function»`.
 
-```shell
-# Running all the tests
-docker compose run --rm api pytest
-
-# Running a specific test
-docker compose run --rm api pytest tests/test_module.py
-
-# Running a specific test function
-docker compose run --rm api pytest tests/test_module.py::test_function
-```
 
-### Running the migrations
-To use morpheus-data image to run the migrations, you need to create a secrets.env file in the morpheus-server 
+## Running the migrations
+To use `morpheus-data` image to run the migrations, you need to create a secrets.env file in the morpheus-server 
 directory. For more information, you can read the morpheus-data [README](./morpheus-data/README.md).
 
-```shell
-# Create migration
-docker-compose run --rm datalib alembic revision --autogenerate -m "Initial migration"
-
-# Migrate / Update the head
-docker-compose run --rm datalib alembic upgrade head
-```
-
-### PG admin
-
-PGadmin is available in: localhost:8002
+* To create a migration: `docker-compose run --rm datalib alembic revision --autogenerate -m "Initial migration"`.
+* To migrate or update to the head: `docker-compose run --rm datalib alembic upgrade head`.
```

Comment:
@asanchezyali please change from inline-code to block-code(with three quote marks). It allows one to copy the command more easily, especially for long commands. Please do the same in other parts of the REAMDE where it is required